### PR TITLE
Docs: Stable Diffusion pipeline

### PR DESCRIPTION
**Pull request by pcuenca**
Part of #293

### CODE REVIEW

**Comment by patrickvonplaten** [comment]
Should we add the other stable diffusion pipelines in this PR as well?

**Review by patrickvonplaten — APPROVED** [review]

File: src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py [hunk 1]
```diff
@@ -62,6 +87,49 @@ def __call__(
         return_dict: bool = True,
         **kwargs,
     ):
+        r"""
```
Comment (patrickvonplaten) [inline src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py:90]: Can we maybe give `enable_attention_slicing` also a docstring ? as well as the disable function?

**Comment by pcuenca** [comment]
> Should we add the other stable diffusion pipelines in this PR as well?

Makes sense!

**Comment by patrickvonplaten** [comment]
Really nice!

**Pull request merged**